bugbot/crash/analyzer.py

```diff
@@ -477,6 +477,41 @@ def is_near_allocator_related_crash(self) -> bool:
         """
         return self.is_near_allocator_crash or self.is_potential_near_allocator_crash
 
+    @property
+    def is_content_crash(self) -> bool:
+        """Whether the crash is related to content process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "content":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_gpu_crash(self) -> bool:
+        """Whether the crash is related to GPU."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "gpu":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_parent_crash(self) -> bool:
+        """Whether the crash is related to the parent process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "parent":
+                return row["count"] > 0
```

Comment:
Kind of related to the above, is this value ever 0? Maybe it makes sense to continue and keep looking rather than returning false if the count is 0?

Reply:
> is this value ever 0? 

I think no. If there are no cases, the type should not show up in the first place. I just used the same approach used in Socorro in case that there are edge cases: https://github.com/mozilla-services/socorro/blob/ff8f5d6b41689e34a6b800577d8ffe383e1e62eb/webapp/crashstats/crashstats/utils.py#L183-L187C21

Reply:
That makes sense.